fMRIVolume/scripts/OneStepResampling.sh

```diff
@@ -108,23 +140,28 @@ echo "date: `date`" >> $WD/log.txt
 echo " " >> $WD/log.txt
 
 
-########################################## DO WORK ########################################## 
+########################################## DO WORK ##########################################
+
+if [ `${FSLDIR}/bin/imtest ${BiasField}` -eq 0 ]; then
+  verbose_red_echo "---> Missing Bias Field image, creating a fake one"
+  ${FSLDIR}/bin/fslmaths ${T1wImage} -abs -add 1 -bin ${BiasField}
```

Comment:
I think the way we usually accomplish this is `-mul 0 -add 1`

Reply:
Fixed.